Gemfile

```diff
@@ -28,6 +28,9 @@ gem 'omniauth-google-oauth2'
 
 # Using ActiveAdmin
 gem 'activeadmin', github: 'activeadmin'
+# Use sidekiq for multithreading
+gem 'sidekiq'
+gem 'redis-rails'
```

Comment:
I've never used this redis-rails gem. Not sure what extra support gives. Having redis installed in your ubuntu should be enought. Why did you add it? Did'nt work without it?